scripts/README.md

```diff
@@ -106,11 +106,15 @@ please do not add new entries to these files. PRs removing (the need for) entrie
 - `count-trans-deps.py`, `import-graph-report.py` and `import_trans_difference.sh` produce various
   summaries of changes in transitive imports that the `PR_summary` message incorporates.
 - `zulip_emoji_merge_delegate.py` is called
-  * every time a `bors d`, `bors merge` or `bors r+` comment is added to a PR and
-  * on every push to `master`.
+  * every time a `bors d`, `bors merge` or `bors r+` comment is added to a PR,
```

Comment:
```suggestion
  * every time a `bors d`, `bors merge` or `bors r` comment is added to a PR,
```
if I am following the text correctly: `bors r+` and `bors r-` both trigger the action.